### ISSUE TITLE
error transpiling https://github.com/silverbulletmd/silverbullet

### ISSUE THREAD

**Issue by kjk** [futurGH/ts-to-jsdoc#30]
I checked out https://github.com/silverbulletmd/silverbullet at version `0034b4e8a7d2efd90ac1056a145865725f869b56` and ran:

```
silverbullet> ts-to-jsdoc.cmd .\common\
ManipulationError: Manipulation error: A syntax error was inserted.

manifest.ts-to-jsdoc.ts:73:2 - error TS1109: Expression expected.

73  *   syntax?: { [key: string]: NodeDef };
... and more error output
```

I imagine this could be newer TypeScript. silverbullet uses deno which uses typescript **5.2.2** in latest version while `ts-to-jsdoc` seems to use **^4.9.4**

**Comment by futurGH** [comment]
Thanks for the issue! Seems to just be an issue with comments in object type aliases. Repro:
```ts
type Test = {
    /* comment */
    prop: string;
}
```
which gets transpiled to the invalid
```js
/**
 * @typedef {{
 *     /* comment */
 *     prop: string;
 * }} Test
 */

**Comment by futurGH** [comment]
Fixed in 1.5.0!

**Comment by kjk** [comment]
Thanks!
